catalogs/hadoop-common/src/main/java/org/apache/gravitino/catalog/hadoop/fs/FileSystemProvider.java

```diff
@@ -65,6 +65,23 @@ public interface FileSystemProvider {
   FileSystem getFileSystem(@Nonnull Path path, @Nonnull Map<String, String> config)
       throws IOException;
 
+  /**
+   * Get the FileSystem instance according to the configuration map and file path.
+   *
+   * @param config The configuration for the FileSystem instance.
+   * @param path The path to the file system.
+   * @param disableCache Whether to cache the FileSystem instance.
+   * @return The FileSystem instance.
+   * @throws IOException If the FileSystem instance cannot be created.
+   */
+  default FileSystem getFileSystem(
+      @Nonnull Path path, @Nonnull Map<String, String> config, boolean disableCache)
+      throws IOException {
+    // disable cache
+    config.put(String.format("fs.%s.impl.disable.cache", scheme()), String.valueOf(disableCache));
```

Comment:
What if the config already contains key `fs.%s.impl.disable.cache`? will we cover it?

Reply:
It will overwrite. I think we should prioritize the disableCache parameter.